fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpIdentifierConverter.java

```diff
@@ -99,8 +98,13 @@ protected String doForward(final String httpUri) {
         throw new IllegalArgumentException("Cannot translate NULL path");
     }
 
-    @Override
-    protected String doBackward(final String fedoraId) {
+    /**
+     * Convert an internal identifier to an external URI.
+     *
+     * @param fedoraId the internal identifier.
+     * @return the external URI.
+     */
+    protected String toExternalId(final String fedoraId) {
         LOGGER.debug(String.format("Translating Fedora ID %s to Http URI", fedoraId));
```

Comment:
I know its not part of this PR, but it'd still be best to switch to using `{}` notation for this log statement and the other ones in this file (its a minor performance issue)